core/account-id/src/lib.rs

```diff
@@ -142,17 +166,22 @@ impl AccountId {
     /// use near_account_id::AccountId;
     ///
     /// let alice: AccountId = "alice.near".parse().unwrap();
-    /// assert!(!alice.is_implicit());
+    /// assert!(!alice.is_near_implicit());
     ///
     /// let rando = "98793cd91a3f870fb126f66285808c7e094afcfc4eda8a970f6648cdf0dbd6de"
     ///     .parse::<AccountId>()
     ///     .unwrap();
-    /// assert!(rando.is_implicit());
+    /// assert!(rando.is_near_implicit());
     /// ```
-    pub fn is_implicit(&self) -> bool {
+    pub fn is_near_implicit(&self) -> bool {
         self.len() == 64 && self.as_bytes().iter().all(|b| matches!(b, b'a'..=b'f' | b'0'..=b'9'))
     }
 
+    /// Returns `true` if this `AccountId` is either NEAR implicit address or ETH-format address.
+    pub fn is_ximplicit(&self) -> bool {
```

Comment:
what does 'x' mean?

Reply:
It is a temporary change to mark all places that used `is_implicit`.
It will become `is_implicit` again before I mark this PR as not draft PR.